subprojects/maven/src/main/java/org/gradle/api/publish/maven/tasks/PublishToMavenRepository.java

```diff
@@ -130,6 +130,12 @@ private CredentialListener credentialListener() {
     }
 
     private boolean areCredentialsSafe(String identity, Credentials toCheck) {
+        // TODO:RC not using a provider does not necessarily imply unsafe credentials
+        // https://github.com/gradle/gradle/issues/22618
+        return true /* isUsingCredentialsProvider(identity, toCheck) */;
```

Comment:
💄 I'd remove the commented out function call

Reply:
Tried to leave a breadcrumb so a reader can figure out why we have a (now) unused private function below

Reply:
Ok, no strong feelings either way.